src/PInvoke.sln

```diff
@@ -142,6 +142,10 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Ole32", "Ole32\Ole32.csproj
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Ole32.Tests", "Ole32.Tests\Ole32.Tests.csproj", "{8B01C254-2EF7-4724-AFD7-FAD39BE69618}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Cabinet", "Cabinet\Cabinet.csproj", "{966BE4C3-657F-4D2C-8717-C04804064186}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Cabinet.Tests", "Cabinet.Tests\Cabinet.Tests.csproj", "{12C7FC54-61AA-413E-A53F-B7AD5D738C70}"
+EndProject
```

Comment:
Good. But don't forget the [rest of these steps]

https://github.com/dotnet/pinvoke/blob/b8977c166596fa8e27df510a77927324a48d2907/templates/AddNewLibrary.ps1#L53-L55